.circleci/config.yml

```diff
@@ -1,13 +1,5 @@
 version: 2
 
-references:
-  container_config: &container_config
-    docker:
-      - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
-        environment:
-          GITHUB_TEAM_NAME_SLUG: laa-get-access
-          APPLICATION_DEPLOY_NAME: laa-fala-webapp
```

Comment:
I assume `GITHUB_TEAM_NAME_SLUG` and `APPLICATION_DEPLOY_NAME` have been moved to CircleCI environment variables.

Do we want to do that? In this case, it seems better to be explicit about it.

Reply:
They're not used anywhere that I can find so they've been removed.